docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayFixed.xml

```diff
@@ -197,6 +203,19 @@
           significant impact on performance.</entry>
         </row>
 
+        <row>
+          <entry><emphasis>sourcePlane</emphasis></entry>
+
+          <entry>The name of the landing zone containing the file</entry>
+        </row>
+
+        <row>
+          <entry><emphasis>destinationNumParts</emphasis></entry>
+
+          <entry>Override the number of parts to be created when
+          spraying.</entry>
+        </row>
```

Comment:
If I know well the handling of this "destinationNumParts" is not fully implemented yet. (At least I can't find any further reference than pass that parameter from ECL to File Services in 8.10.x)
I think it would be good to clarify it with @ghalliday to avoid any further confusion. 
(This comment apply all further occurrences of this parameter)

Reply:
destinationNumParts is documented in File.ecl's JavaDoc comments. 
The question of deprecating the sourceIP/destinationIP is really outside the scope of this PR. Shouldn't we open a seperate Jira for that? We should, obviously, continue support for backward compatibility.